documentation/modules/ROOT/pages/connectors/db2.adoc

```diff
@@ -0,0 +1,1503 @@
+The client applications read the Kafka topics that correspond to the database tables they're interested in following, and react to every row-level event it sees in those topics.
+
+The database administrator normally enables _CDC_ in the middle of the life of a table.
+This means that the connector won't have the complete history of all changes that have been made to the table.
+Therefore, when the Db2 connector first connects to a particular Db2 database, it starts by performing a _consistent snapshot_ of each of the whitelisted tables.
+After the connector completes the snapshot, it continues streaming changes from the exact point at which the snapshot was made for tables in capture mode.
+This way, we start with a consistent view of all of the data, yet continue reading without having lost any of the changes made while the snapshot was taking place.
+
+The connector is also tolerant of failures.
+As the connector reads changes and produces events, it records the position in the database log (_LSN / Log Sequence Number_) of the _CDC_ record for each event.
+If the connector stops for any reason (including communication failures, network problems, or crashes), upon restart it simply continues reading the _CDC_ tables where it last left off.
+This includes snapshots: if the snapshot was not completed when the connector is stopped, upon restart it will begin a new snapshot. Care needs to be taken with mixing multiple snapshot in the same topic as delete operations may be missed.
+
+[[setting-up-Db2]]
+== Setting up Db2 
+
+Before using the Db2 connector to monitor the changes committed on the Db2 database, first enable _CDC_ on a monitored database.
+
+We recommend using the UDF functions that we provide.
+However these are only for convenience and manually performing the same commands using
+db2 control commands would have the same effect. 
+
+We assume here that the contents of 
+debezium-connector-db2/src/test/docker/db2-cdc-docker are available
+on the machine on which Db2 is running in $HOME/asncdctools/src
+and that the user has logged-in as the db2inst1 user.
+
+First compile this function on the Db2 server, using the _bldrtn_
+C-compiler
+
+[source,shell]
+
+----
+$ cd $HOME/asncdctools/src
+----
+
+[source,shell]
+
+----
+$ ./bldrtn asncdc
+----
+
+
+Start the database if not already running:
+
+
+[source,shell]
+
+----
+$ db2 start db <DB NAME>
+
+----
+
+We need to make sure that the metadata catalog can be read via JDBC
+
+[source,shell]
+----
+$ cd $HOME/sqllib/bnd
+----
+
+[source,shell]
+----
+$ db2 bind db2schema.bnd blocking all grant public sqlerror continue 
+----
+
+The database needs to have been recently backed-up so doing following if this is not the case:
+
+[source,shell]
+----
+$ db2 backup db <DB NAME> to /dev/null
```

Comment:
Having no experience with that tool: what does the "to /dev/null" apply to? Any log messages, or is it sort of a joke as target for the backup? In that case I'd prefer to remove it so to avoid any surprises by unknowing users :)

Reply:
I added some more explanation in the text